llvm/test/CodeGen/RISCV/rvv-cfi-info.ll

```diff
@@ -56,10 +56,11 @@ define riscv_vector_cc <vscale x 1 x i32> @test_vector_callee_cfi(<vscale x 1 x
 ; OMIT-FP-NEXT:    .cfi_def_cfa sp, 16
 ; OMIT-FP-NEXT:    .cfi_restore v1
 ; OMIT-FP-NEXT:    .cfi_restore v2
+; OMIT-FP-NEXT:    .cfi_restore v3
 ; OMIT-FP-NEXT:    .cfi_restore v4
-; OMIT-FP-NEXT:    .cfi_restore v1
-; OMIT-FP-NEXT:    .cfi_restore v2
-; OMIT-FP-NEXT:    .cfi_restore v4
+; OMIT-FP-NEXT:    .cfi_restore v5
+; OMIT-FP-NEXT:    .cfi_restore v6
+; OMIT-FP-NEXT:    .cfi_restore v7
```

Comment:
I'm not very familiar with the V extension, but this looks much better, if `vl2r.v` and `vl4r.v` are going to restore 2 and 4 consecutive v registers respectively.